lib/doekbase/data_api/tests/examples/proteins_to_fasta.py

```diff
@@ -0,0 +1,47 @@
+
+__author__ = 'Marcin Joachimiak <mjoachimiak@lbl.gov>'
+__date__ = '11/06/15'
+
+import doekbase.data_api
+from doekbase.data_api.annotation.genome_annotation.api import GenomeAnnotationAPI
+import os
+
+def get_aliases (data):
```

Comment:
proteins_to_fasta.py get_aliases() could be eliminated in place of " ".join(geneinfo["protein_aliases"]) on line 24